db/migrate/20170710145821_add_confirmable_to_user.rb

```diff
@@ -0,0 +1,16 @@
+class AddConfirmableToUser < ActiveRecord::Migration
+  def up
+    add_column :spree_users, :confirmation_token, :string
+    add_column :spree_users, :confirmed_at, :datetime
+    add_column :spree_users, :confirmation_sent_at, :datetime
+    add_column :spree_users, :unconfirmed_email, :string
+    add_index :spree_users, :confirmation_token, :unique => true
+
+    # Set all current users to confirmed
+    Spree::User.update_all(confirmed_at: Time.zone.now)
```

Comment:
I thought we were going to try and preserve our existing knowledge of confirmed and unconfirmed email addresses to populate `confirmed_at` for users? Or did we change our minds?

````ruby
confirmed_emails = Enterprise.confirmed.pluck(:email).uniq
Spree::User.where(email: confirmed_emails).update_all(confirmed_at: Time.zone.now)
````

Reply:
I think there was a discussion about it elsewhere and this was what was decided.

Reply:
Hm, ok. It feels like potentially valuable data and once it's gone, it's gone. We can't get it back. Do you remember where the discussion happened?

Reply:
I initially thought it would be good to preserve that data as well. The discussion is in the comments here: https://github.com/openfoodfoundation/openfoodnetwork/pull/1768

Reply:
Thanks. Looks like we're outnumbered. :)